fuzz/helper.py

```diff
@@ -9,13 +9,16 @@
 
 """Fuzzing helper, creates and uses corpus/crash directories.
 
-fuzzer.py <fuzzer> <extra fuzzer arguments>
+helper.py <fuzzer> <extra fuzzer arguments>
```

Comment:
Wanted to point out the addition made here to fix the name for `fuzzer.py` to `helper.py` to match the name of the file.